### PR TITLE
Feature/admin log

### PR DESCRIPTION
**Pull request by celynolivia**
Insert Log (EDIT, DELETE, RESEND),
Show List Admin Log,
Filter

### CODE REVIEW

File: src/pages/ActivityLog.js [hunk 1]
```diff
@@ -1,13 +1,160 @@
...
+      let fixdate = year + "-" + month + "-" + day;
+      setDate(fixdate);
+    }
+    else{
+        setDate(null);
+    }
+  }
+
+  function onChange(pageNumber) {
+    console.log(pageNumber)
```
Comment (yulyulyharuka) [inline src/pages/ActivityLog.js]: kindly delete this console log

File: src/pages/Customers.js [hunk 2]
```diff
@@ -116,6 +116,30 @@ export default function Customers() {
...
+      //success
+      console.log(res,"success")
+      
+    }).catch((err) => {
+      //failed
+      console.log(err,"error")
+      
+    }).finally(() => {
+
+    })
```
Comment (yulyulyharuka) [inline src/pages/Customers.js]: error handling when can not log

**Review by yulyulyharuka — CHANGES_REQUESTED** [review]

File: src/pages/ActivityLog.js [hunk 3]
```diff
@@ -1,13 +1,160 @@
...
+    }
+    
+    if(date == null && username == null){
+      url = "http://localhost:8000/v2/log/"+pageNumber
+    }else if(date != null && username == null){
+      url = "http://localhost:8000/v2/log/d/"+modifDate+"/"+pageNumber;
+    }else if(date == null && username != null){
+      url = "http://localhost:8000/v2/log/u/"+username+"/"+pageNumber;
+    }else if(date != null && username != null){
+      url = "http://localhost:8000/v2/log/"+username+"/"+modifDate +"/" + pageNumber;
```
Comment (polipopoliko) [inline src/pages/ActivityLog.js]: optional: change the host number to config.apiHost + YOUR_ENDPOINT

File: src/pages/ActivityLog.js [hunk 4]
```diff
@@ -1,13 +1,160 @@
...
+        return singleRow
+      })
+      setList(tableData)
+      setTotaldata(res.data.data.count)
+    }).catch((err) => {
+      //failed
+      console.log(err,'error')
+    }).finally(() => {
+      console.log("finally")
+      // setLoading(false);
```
Comment (polipopoliko) [inline src/pages/ActivityLog.js]: error handling?

**Review by polipopoliko — CHANGES_REQUESTED** [review]

File: src/pages/ActivityLog.js (hunk 1 above)
Comment (celynolivia) [reply src/pages/ActivityLog.js]: okaayy

**Review by yulyulyharuka — APPROVED** [review]

**Pull request merged**